core/src/main/java/io/confluent/kafka/schemaregistry/rest/resources/SubjectsResource.java

```diff
@@ -67,6 +70,8 @@ public void lookUpSchemaUnderSubject(final @Suspended AsyncResponse asyncRespons
                                        final @HeaderParam("Content-Type") String contentType,
                                        final @HeaderParam("Accept") String accept,
                                        @PathParam("subject") String subject,
+                                       @QueryParam("deleted") boolean
```

Comment:
Yeah, I guess it's just the implicit default. Might be nice for a reader of the code for it to be explicit, but it's fine if it is tested.